mod/InGameTracker/TrackerManager.cs

```diff
@@ -0,0 +1,212 @@
+
+        /// <summary>
+        /// A list of hints received from the AP server. In order, the strings are the item name, the location, and the world.
+        /// </summary>
+        public Dictionary<string, Tuple<string, string>> Hints;
+
+        // This dictionary is the list of items in the Inventory Mode
+        // They'll also display in this order, with the second string as the visible name
+        public readonly Dictionary<string, string> ItemEntries = new()
+        {
+            {Item.Coordinates.ToString(), "Eye of the Universe Coordinates" },
+            {Item.LaunchCodes.ToString(), "Launch Codes"},
+            {Item.Translator.ToString(), "Translator" },
+            {Item.Signalscope.ToString(), "Signalscope" },
+            {Item.EntanglementRule.ToString(), "Suit Lights Controls" },
+            {Item.ElectricalInsulation.ToString(), "Electrical Insulation" },
+            {Item.SilentRunning.ToString(), "Silent Running Mode" },
+            {Item.TornadoAdjustment.ToString(), "Tornado Aerodynamic Adjustments" },
+            {Item.Scout.ToString(), "Camera: Scout Launcher" },
+            {Item.CameraGM.ToString(), "Camera: Ghost Matter Frequency" },
+            {Item.CameraQuantum.ToString(), "Camera: Quantum Objects" },
+            {Item.WarpPlatformCodes.ToString(), "Nomai: Warp Platform Codes" },
+            {Item.WarpCoreManual.ToString(), "Nomai: Warp Core Installation Manual" },
+            {Item.ShrineDoorCodes.ToString(), "Nomai: Shrine Door Codes" },
+            {"FrequencyOWV", "Frequency: Outer Wilds Ventures" },
+            {Item.FrequencyDB.ToString(), "Frequency: Distress Beacons" },
+            {Item.FrequencyQF.ToString(), "Frequency: Quantum Fluctuations" },
+            {Item.FrequencyHS.ToString(), "Frequency: Hide and Seek" }
+        };
+
+        private ICustomShipLogModesAPI api;
+        private TrackerInventoryMode inventoryMode;
+        private ArchipelagoSession session;
+
+        private void Awake()
+        {
+            api = Randomizer.Instance.ModHelper.Interaction.TryGetModApi<ICustomShipLogModesAPI>("dgarro.CustomShipLogModes");
+            if (api != null )
+            {
+                Randomizer.OWMLModConsole.WriteLine("Custom Ship Log Modes API found!", OWML.Common.MessageType.Success);
+            }
+            else
+            {
+                Randomizer.OWMLModConsole.WriteLine("Custom Ship Log Modes API not found! Make sure the mod is correctly installed. Tracker will not function.", OWML.Common.MessageType.Error);
+                return;
+            }
+
+            inventoryMode = gameObject.AddComponent<TrackerInventoryMode>();
+            LoadManager.OnCompleteSceneLoad += (scene, loadScene) =>
+            {
+                if (loadScene == OWScene.SolarSystem) AddModes();
+            };
+
+            NewItems = new();
+            foreach (string item in ItemEntries.Keys)
+            {
+                NewItems.Add(item, false);
```

Comment:
Maybe `ItemIsNew`? The name `NewItems` had me expecting a list or set of string ids, not an id->bool map.

Reply:
Sure. This will be one of the Dictionaries removed when I change how the ItemEntries work.